core/primitives-core/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "near-primitives-core"
+version = "0.1.0"
+authors = ["Near Inc <hello@nearprotocol.com>"]
+edition = "2018"
+license = "Apache-2.0"
+repository = "https://github.com/nearprotocol/nearcore"
+homepage = "https://github.com/nearprotocol/nearcore"
+description = """
+This crate provides the core set of primitives used by other nearcore crates including near-primitives
+"""
+
+[dependencies]
+base64 = "0.11"
+borsh = "0.8.1"
+bs58 = "0.4"
+hex = "0.4"
+derive_more = "0.99.3"
+num-rational = { version = "0.3.1", features = ["serde"]}
+serde = { version = "1", features = ["derive"] }
+serde_json = "1"
+sha2 = ">=0.8,<0.10"
```

Comment:
why

Reply:
Not sure, I just copied this from `primitives`.